test/functional/test-runner.js

```diff
@@ -0,0 +1,94 @@
+var harness      = require('./harness');
+
+
+function checkTestByBrowserAlias (testName, browserAlias) {
+    var matchesOnly = testName.match(/\[ONLY\:([,\w]*)\]/);
+    var matchesSkip = testName.match(/\[SKIP\:([,\w]*)\]/);
+
+    var only = true;
+    var skip = false;
+
+    if (matchesOnly !== null)
+        only = matchesOnly[1].indexOf(browserAlias) > -1;
+
+    if (matchesSkip !== null)
+        skip = matchesSkip[1].indexOf(browserAlias) > -1;
+
+    return only && !skip;
+}
+
+function filterErrorsByUserAgent (userAgents, errors) {

```

Comment:
let's change the arguments order `(errors, userAgents)` because of the name of the function (errors by useragents)